java/ql/src/experimental/Security/CWE/CWE-532/SensitiveInfoLog.ql

```diff
@@ -9,21 +9,21 @@
 
 import java
 import semmle.code.java.dataflow.TaintTracking
+import semmle.code.java.security.SensitiveActions
 import DataFlow
 import PathGraph
 
 /**
- * Gets a regular expression for matching names of variables that indicate the value being held is a credential
+ * Gets a regular expression for matching names of variables that indicate the value being held may contain sensitive information
  */
 private string getACredentialRegex() {
-  result = "(?i).*challenge|pass(wd|word|code|phrase)(?!.*question).*" or
-  result = "(?i)(.*username|.*secret|url).*"
+  result = "(?i)(.*uri|url).*"
```

Comment:
Where did `.*uri` come from? In this PR we should keep SensitiveInfoLog.ql's behaviour unchanged

Reply:
Thanks @smowton. I've updated the regex to remove <code>.*uri</code>, which is similar to <code>url</code> but not identical.